dandi/download.py

```diff
@@ -1035,3 +1032,15 @@ def feed(self, path: str, status: dict) -> Iterator[dict]:
             lgr.warning(
                 "Unexpected download status dict for %r received: %r", path, status
             )
+
+
+def multiasset_target(url_path: str, asset_path: str) -> str:
+    """
+    When downloading assets for a `MultiAssetURL` with `~MultiAssetURL.path`
+    equal to ``url_path``, calculate the path (relative to the output path) at
+    which to save the asset with path ``asset_path``.
+    """
+    prefix = op.dirname(url_path.rstrip("/"))
+    if prefix:
+        prefix += os.sep
+    return asset_path[len(prefix) :]
```

Comment:
```suggestion
    assert(asset_path.startswith(prefix))
    return asset_path[len(prefix) :]
```
correct? possibly relates to the lstripping above.